app/service/rest.py

```diff
@@ -475,10 +475,18 @@ def search_for_notification_by_to_field(service_id, search_term, statuses, notif
         service_id=service_id,
         search_term=search_term,
         statuses=statuses,
-        notification_type=notification_type
+        notification_type=notification_type,
+        page=1,
+        page_size=current_app.config['PAGE_SIZE'],
     )
     return jsonify(
-        notifications=notification_with_template_schema.dump(results, many=True).data
+        notifications=notification_with_template_schema.dump(results.items, many=True).data,
+        links=pagination_links(
```

Comment:
Optional: We have 4 other places we are returning paginated results. Three of the four seem to also return keys for total and page_size. Would be good to make it consistent I think?

Do we also want a unit test to check one of the responses at least contains the links key? I'm not sure if I'm too interested in testing the values of it as don't want to retest pagination_links itself but just as an indicator that we are returning paginated results?